distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/SampleNsiliClient.java

```diff
@@ -334,34 +334,25 @@ public void downloadProductFromDAG(DAG dag) {
 
                 final String productPath = "product.jpg";
                 LOGGER.info("Downloading product : {}", fileDownloadUri);
-                BufferedInputStream inputStream = null;
-                FileOutputStream outputStream = null;
                 try {
-                    inputStream = new BufferedInputStream(fileDownloadUri.toURL()
-                            .openStream());
-                    outputStream = new FileOutputStream(new File(productPath));
-
-                    final byte data[] = new byte[1024];
-                    int count;
-                    while ((count = inputStream.read(data, 0, 1024)) != -1) {
-                        outputStream.write(data, 0, count);
-                    }
+                    try (BufferedInputStream inputStream = new BufferedInputStream(fileDownloadUri.toURL()
+                            .openStream())) {
+                        try (FileOutputStream outputStream = new FileOutputStream(new File(
```

Comment:
Does this need to be nested?  Wondering if we can combine the two streams in the same try-with-resources

Reply:
ah yeah, I think I can combine those. I was individually tracking those on SonarQube. 👍